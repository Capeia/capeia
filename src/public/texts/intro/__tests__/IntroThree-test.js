// @flow
/* eslint-env jest, jasmine */

import React from 'react'
import renderer from 'react-test-renderer'

describe('<IntroThree>', () => {
  it('renders correctly', () => {
    jest.deepUnmock('public/texts/intro/IntroThree')
    const Component = require('public/texts/intro/IntroThree').default
    const tree = renderer.create(<Component />).toJSON()
    expect(tree).toMatchSnapshot()
  })
})
