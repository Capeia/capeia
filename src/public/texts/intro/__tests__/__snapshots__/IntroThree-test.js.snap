// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<IntroThree> renders correctly 1`] = `
<div
  className="container-fluid"
>
  <div
    className="row"
  >
    <div
      className="contentContainer  col-lg-6 col-lg-offset-3 col-md-8"
    >
      <div
        className={undefined}
      >
        <div
          className=""
        >
          <h1>
            Our Motivation
          </h1>
          <p>
            Imagine being a student of mathematics and astronomy in the year 1440 CE. Where would you have to go to get the best education and carry out cutting edge research in your field? You may find the answer astonishing: you would need to go to Samarkand and apply to the madrasa of Mirza Muhammad Taraghay, better known as Ulugbheg. It was there in today’s Uzbekistan that the best equipped observatory of its time was located and there that, in a lively intellectual climate
            <span
              className="ImageDecorator icon"
              onClick={[Function]}
              style={
                Object {
                  "backgroundImage": "url(article/glyphicon-pushpin.png)",
                }
              }
            >
              　
            </span>
            , students attended lectures and courses given by the most progressive minds of their time. At the heart of this multi-million dollar
            <span
              className="ImageDecorator icon"
              onClick={[Function]}
              style={
                Object {
                  "backgroundImage": "url(article/glyphicon-pushpin.png)",
                }
              }
            >
              　
            </span>
            observatory was an enormous and most skilfully manufactured sextant of brass with a radius of more than 40 m. This device allowed astronomical measurements to be made with unprecedented precision. Ulughbeg’s star catalog, the Zij-i Sultani, was only surpassed 150 years later by Tycho Brahe’s Star Catalog D, and Ulughbeg’s determination of the earth’s axial tilt remained valid until the modern era. By 1449, however, Ulughbeg had been murdered
            <span
              className="ImageDecorator icon"
              onClick={[Function]}
              style={
                Object {
                  "backgroundImage": "url(article/glyphicon-pushpin.png)",
                }
              }
            >
              　
            </span>
            , his observatory torn down, the curriculum of his madrasa purged of all science, and his faculty staff dispersed. The end of Ulughbeg’s madrasa and of the Samarkand observatory also marked the ultimate end of what has become known as the Islamic Golden Age. From then on, orthodoxy and traditionalism prevailed and education was narrow and based on authority rather than on reason.
          </p>
          <div>
            <img
              src="intro/ulugh_beg_observatory.jpg"
            />
            <div
              className="imageDescription"
            >
              <span>
                All that remains of Ulugh Beg’s observatory is the foundation and the lower, subterranean part of the huge sextant. Excavated by Russian archaeologists in 1908, the site is now part of a museum in Samarkand, Uzbekistan
                 
                <em>
                  (Image credit: Alaexis, CC-BY-SA, Wikimedia Commons)
                </em>
                .
              </span>
            </div>
          </div>
          <p>
            So did Christian Europe then feel free to take over science and waft away to higher levels of existence? Not quite so. Galileo Galilei, the founder of modern experimental science, was hauled before the Inquisition in 1633 and charged with heresy for supporting the idea of heliocentrism. He was shown the instruments of torture and forced to recant his view that the earth moves around the sun. He was placed under house arrest for the rest of his life and was forbidden to publish. He had to consider himself lucky, however. Giordano Bruno was burnt at the stake in 1600 for claiming that there were many other worlds, apart from the earth. Today, one probably would refer to "exoplanets" and find this concept quite intriguing.
          </p>
          <p>
            So what was it that stifled the scientific and technical progress of the Islamic world, and almost did the same in Christian Europe? Presumably it was not a lack of political (i.e. military) success. The Ottoman Empire conquered Constantinople in 1453, a huge victory over Christian Europe but it could not translate this into any form of widespread cultural predominance. Neither is a lack of material wealth a plausible explanation for the gradual decline of the Islamic world. The wealth and prosperity of the Mughal Empire is legendary but little more than the Taj Mahal remains today as a testimony to this empire’s former greatness. In 1517, however, something extraordinary happened in Europe that gained momentum and would radically change the mindscape and pave the way for the scientific and technical revolution that was to become the hallmark of Western civilization. A catholic friar by the name of Martin Luther fundamentally criticized the Roman Catholic Church and challenged the authority of the pope
            <span
              className="ImageDecorator icon"
              onClick={[Function]}
              style={
                Object {
                  "backgroundImage": "url(article/glyphicon-pushpin.png)",
                }
              }
            >
              　
            </span>
            . In actual fact he was not the first person to do so. In 14th century England John Wycliffe had already taken on the pope and in the early 15th century the Czech reformer Jan Hus argued for a thorough reformation of the Catholic Church. This is the more remarkable because any criticism of the Church, and especially of the pope, was potentially life-threatening in those days
            <span
              className="ImageDecorator icon"
              onClick={[Function]}
              style={
                Object {
                  "backgroundImage": "url(article/glyphicon-pushpin.png)",
                }
              }
            >
              　
            </span>
            . It does illustrate though, that at this time in Europe discontent with the secular and spiritual power of the Church was growing. Change was in the air and Luther was the man to crystallize it. It is important to realize that in the Medieval period the Church was much more influential in the Western world than it is today. Its influence went far beyond the spiritual realm by being deeply intertwined with secular power. From an intellectual point of view, the Church kept it rather simple: the bottom line was that everything of relevance to human life was already known and contained in the Christian Scriptures. What was not mentioned or addressed therein was simply not relevant and musing about it was a waste of time, at best. Luther was an intellectual though and he was not willing to accept this simple doctrine. Instead he developed a concept of two governments or kingdoms, one for worldly, secular issues and one for heavenly issues. Although he still saw God as the one who ordained both governments, this was the first step towards the development of a distinctly secular world order. The magnitude of this tectonic shift in thinking cannot be overestimated. It came at the huge cost of several million lives over the course of the following 130 years, but ultimately it liberated the mind from any ideological - but not moral - constraints and unleashed enormous creative and imaginative power. While your heart might still be occupied with faith, your mind was free to deal with earthly issues
            <span
              className="ImageDecorator icon"
              onClick={[Function]}
              style={
                Object {
                  "backgroundImage": "url(article/glyphicon-pushpin.png)",
                }
              }
            >
              　
            </span>
            . Whether the earth rotates around the sun or whether it is the other way round was not an issue that the reformed Church did pay much attention to
            <span
              className="ImageDecorator icon"
              onClick={[Function]}
              style={
                Object {
                  "backgroundImage": "url(article/glyphicon-pushpin.png)",
                }
              }
            >
              　
            </span>
            .
          </p>
          <div>
            <img
              src="intro/luther_memorial_worms.jpg"
            />
            <div
              className="imageDescription"
            >
              <span>
                The Martin Luther Memorial in Worms, Germany. At the base of Luther’s central statue credit is given to earlier Church reformers including John Wycliffe and Jan Hus. Luther is flanked by local rulers who provided shelter and support when he was declared an outlaw by the Emperor in 1521
                 
                <em>
                  (Image credit: US Library of Congress)
                </em>
                .
              </span>
            </div>
          </div>
          <p>
            So are we safe now? Can we lean back and ponder the big questions of mankind, fight disease, and conceive new technologies for a better quality of life on this planet? Be warned by history. A "two-government" doctrine, although not formulated explicitly, was at times also common during the Golden Islamic Age. Islamic empires and dynasties were often characterized by outspoken tolerance towards the faith-versus-reason question. Great minds, including al-Biruni and al-Farabi, propagated the notion that society should be ruled by reason rather than by any faith-related dogmas. This was a truly heretical attitude to have but there was no Inquisition that forced them to recant, nor were they burned at the stake - or executed elsewise. Tolerance, however, does not mean adoption. Unlike in reformed Europe, an enlightened world view was never adopted by any Islamic society as a whole. But even a thoroughly enlightened and, by any standards, modern society is not safe from falling prey to a dark medieval type of doctrine. A human lifespan ago exactly this happened in Germany. Its Jewish community was extinguished, not only physically but also intellectually. "Jewish physics" was forbidden in favour of a pure "Aryan physics"
            <span
              className="ImageDecorator icon"
              onClick={[Function]}
              style={
                Object {
                  "backgroundImage": "url(article/glyphicon-pushpin.png)",
                }
              }
            >
              　
            </span>
            . Nazism was a terrible setback in what one may describe as the evolution of a humanitarian civilization. Unfortunately though, history does not have any inherent direction or meaning. It just happens, as does biological evolution, in an undirected and ultimately unpredictable manner.
          </p>
          <p>
            Are we therefore completely clueless with regard to our destiny? Isn’t there anything we can do to steer the path of our own society in a beneficial way? Fortunately there is. We think the best we can do is to work for an "open society", as defined by Sir Karl Popper. Sir Karl was a man of Jewish descent with a lucid mind who developed an early interest in questions of philosophy and sociology when he was confronted with the rise of Nazism. This not only made him leave his home town of Vienna for the freedom of New Zealand
            <span
              className="ImageDecorator icon"
              onClick={[Function]}
              style={
                Object {
                  "backgroundImage": "url(article/glyphicon-pushpin.png)",
                }
              }
            >
              　
            </span>
            but also to think deeply about how civilizations develop and how and why reactionary movements may occur. In his influential work 
            <em>
              "The open society and its enemies"
            </em>
             he defines key elements of an open society including democracy, secularism, human rights, freedom of the mind, freedom of the word, and freedom of the press, as well as political transparency and tolerance
            <span
              className="ImageDecorator icon"
              onClick={[Function]}
              style={
                Object {
                  "backgroundImage": "url(article/glyphicon-pushpin.png)",
                }
              }
            >
              　
            </span>
            . He maintained that an open society should never close itself to new input because knowledge is never complete, but ongoing. While this all probably sounds reasonable and appealing, Sir Karl emphasized that an open society does not come at no cost. At its center he places the critical and dedicated individual. This in turn places a significant burden on each of us because in an open society the rational individual has to continually make personal decisions and take responsibility.
          </p>
          <div>
            <img
              src="intro/new_zealand.jpg"
            />
            <div
              className="imageDescription"
            >
              <span>
                At the roadside in the Canterbury region on the South Island, New Zealand. In 1937, one year before the annexation of his Austrian home country to Nazi-Germany, Karl Popper emigrated to New Zealand to lecture at the University of Christchurch. It was there, in this remote part of the free world, where he wrote one of his most influential books, 
                <em>
                  “The open society and its enemies”
                </em>
                 
                <em>
                  (Image credit: 
                  <a
                    href="https://flic.kr/p/dJgqRG"
                    target="_blank"
                  >
                    Colin Capelle
                  </a>
                  , CC-BY)
                </em>
                .
              </span>
            </div>
          </div>
          <p>
            Science clearly thrives best in an open society. In fact there is positive feedback involved: an open society is conducive to the development of scientific minds and the acquisition of new knowledge, which in turn strengthens the humanitarian values upon which the open society is based. It is in this very spirit of an open society that 
            <a
              href="http://www.capeia.com"
            >
              capeia.com
            </a>
             has been conceived
            <span
              className="ImageDecorator icon"
              onClick={[Function]}
              style={
                Object {
                  "backgroundImage": "url(article/glyphicon-pushpin.png)",
                }
              }
            >
              　
            </span>
            . We cordially invite you to be critical and dedicated participants.
          </p>
          <p>
            Enjoy it!
          </p>
          <div>
            <h2
              style={
                Object {
                  "textAlign": "center",
                }
              }
            >
              <a
                href="/welcome"
              >
                I
              </a>
               | 
              <a
                href="/welcome/2"
              >
                II
              </a>
               | 
              III
            </h2>
            <div
              className="row"
            >
              <div
                className="col-sm-6"
              >
                <a
                  href="/welcome/2"
                >
                  ‹ What Capeia is not
                </a>
              </div>
              <div
                className="col-sm-6"
                style={
                  Object {
                    "textAlign": "right",
                  }
                }
              />
            </div>
          </div>
        </div>
      </div>
    </div>
    <div
      className="sidebarContainer col-lg-3 col-md-4"
    >
      <aside
        className="sidebar"
      />
    </div>
  </div>
</div>
`;
